### PR TITLE
{vis}[foss/2023a] PyopenCL v2023.1.4

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/p/PyOpenCL/PyOpenCL-2023.1.4-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ('pocl', '4.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('appdirs', '1.4.4', {
+        'checksums': ['7d5d0167b2b1ba821647616af46a749d1c653740dd0d2415100fe26e27afdf41'],
+    }),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyOpenCL/PyOpenCL-2023.1.4-foss-2023a.eb:23]: `appdirs` is provided by `Python-bundle-PyPI/2023.06-GCCcore-12.3.0`, which is a dependency of `SciPy-bundle`, so can be removed here

```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3139.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/4c3163e9fd85edcec094910f6fac20bf for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Pull request merged**